qutebrowser/config/configdata.yml

```diff
@@ -1100,6 +1100,22 @@ hints.mode:
       - word: Use hints words based on the html elements and the extra words.
   desc: Mode to use for hints.
 
+hints.extended_filter_attrs:
+  default:
+    - "text"
+  type:
+    name: List
+    valtype:
+      name: String
+      none_ok: true
+      valid_values:
+        - text: "the actual node text"
+        - title: "the 'title' attribute"
+        - alt: "the 'alt' attribute"
```

Comment:
Could you turn these into sentences? Just have a look at other `valid_values` definitions in this file. This is just for consistency and make hings more self-explanatory. Also, if the text doesn't include any `'`, there is no need to wrap it in `"`s.

Reply:
Is it OK like this?